user_docs/de/userGuide.t2t

```diff
@@ -3272,4 +3272,3 @@ Die folgenden Werte können geändert werden:
 Wenn Sie weitere Informationen oder Hilfe bezüglich NVDA benötigen, besuchen Sie bitte die NVDA-Homepage unter NVDA_URL.
 Neben Ressourcen der Community und dem technischen Support finden Sie auch zusätzliche Dokumentationen.
 Auf diesen Seiten werden Informationen und Ressourcen zur NVDA-Entwicklung ebenfalls bereitgestellt.
-

```

Comment:
Please reintroduce this line - there should be no changes to the German documentation in this PR as it would introduce conflicts for translators.

Reply:
I'm afraid it isn't. This should be just empty line not hyphen. Essentially when looking at the list of files modified by this PR German user guide should not be mentioned at all.

Reply:
So I restored it from nvaccess:master and it's gone from the "files changed" list!